src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -131,6 +135,13 @@ private void emit(CodeWriter codeWriter) throws IOException {
       codeWriter.emit("\n");
     }
 
+    if (!staticImports.isEmpty()) {
+      for (String signature : staticImports) {
```

Comment:
or maybe just use a `TreeSet` for ’em? Order is never important, and that saves a List to Set conversion later!